src/nuclio/projects/project/functions/version/version.component.js

```diff
@@ -159,7 +159,7 @@
 
             lodash.set(ctrl.version, 'spec.build', lodash.merge({
                 image: '',
-                readinessTimeout: 60,
+                readinessTimeoutSeconds: 60,
```

Comment:
These are the defaults for `spec.build`.
Since `readinessTimeoutSeconds` needs to move to `spec` from `spec.build`, this line here should be removed. The default of 60 seconds should be set elsewhere, where the defaults of `spec` are.